src/data_provider/Extractor.py

```diff
@@ -132,3 +149,21 @@ def create_route_to_stops_dict(stop_times_df: pd.DataFrame, route_ids_df: pd.Dat
                 route_to_stops_dict[route_id] = []
             route_to_stops_dict[route_id].append(stop_id)
         return route_to_stops_dict
+
+    @staticmethod
+    def _get_walking_time(lon_1, lat_1, lon_2, lat_2) -> int:
+        lon_1 = radians(lon_1)
+        lon_2 = radians(lon_2)
+        lat_1 = radians(lat_1)
+        lat_2 = radians(lat_2)
+        earth_radius = 6373000.0
+        dlon = lon_2 - lon_1
+        dlat = lat_2 - lat_1
+
+        a = sin(dlat / 2) ** 2 + cos(lat_1) * cos(lat_2) * sin(dlon / 2) ** 2
+        c = 2 * atan2(sqrt(a), sqrt(1 - a))
+
+        distance = earth_radius * c
```

Comment:
we could use an external dependency for such calculations, but that's OK as well